pipelines/build/common/build_base_file.groovy

```diff
@@ -124,12 +124,25 @@ class Builder implements Serializable {
     }
 
     List<String> getTestList(Map<String, ?> configuration) {
-        if (configuration.containsKey("test")) {
+        List<String> nightly = ['sanity.openjdk', 'sanity.system', 'extended.system', 'sanity.perf', 'sanity.external']
```

Comment:
These look like constants to me, should we hoist up to the class level?

Reply:
Yes, updated.